examples/angular/src/app/components/contentlets/contentlets.component.ts

```diff
@@ -1,14 +1,16 @@
 import { Component, Input } from '@angular/core';
 import { Contentlet } from '@dotcms/client/src/lib/client/content/shared/types';
 import { GenericContentlet } from '../../utils';
-import { DatePipe, NgOptimizedImage } from '@angular/common';
+import { DatePipe, NgFor, NgOptimizedImage } from '@angular/common';
```

Comment:
@zJaaal Why add the NgFor?, checking the code is not used.

Reply:
I had some problems while rendering `ng-content` and I tested by using the `NgFor` and forgot to delete it. So yeah I'll remove that import